onnxruntime/python/tools/transformers/models/whisper/whisper_helper.py

```diff
@@ -13,6 +14,8 @@
 import numpy as np
 import torch
 from transformers import WhisperConfig, WhisperForConditionalGeneration, WhisperProcessor
+from whisper import _MODELS, _ALIGNMENT_HEADS, _download
+from whisper.model import Whisper, ModelDimensions
```

Comment:
Since these imports are from [here](https://github.com/openai/whisper) and the default export is the Hugging Face implementation, I think we should put them inside `load_model_openai`. Otherwise, they will be needed even when exporting just the Hugging Face version.